appengine/standard/migration/taskqueue/counter/main-test.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pytest
+import uuid
+
+
+os.environ['LOCATION'] = 'us-central1'
+os.environ['QUEUE'] = str(uuid.uuid4())
```

Comment:
These don't match the variables we asked the user to define

Reply:
How so? These don't set GOOGLE_CLOUD_PROJECT because the test environment already does that.

Reply:
I don't remember what I meant by this, but maybe it was the lack of project, so I suppose we're good